CMakeLists.txt

```diff
@@ -43,6 +43,12 @@ option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
 option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
 option(IREE_BUILD_EXPERIMENTAL "Builds experimental projects." OFF)
 
+#TODO: Make is functional!
+set(IREE_HAL_BACKENDS_TO_BUILD "all"
+  CACHE STRING "Semicolon-separated list of HAL backends to build, or \"all\"." FORCE)
+set(IREE_COMPILER_BACKENDS_TO_BUILD "all"
```

Comment:
Similarly, the compiler backends are usually called target_backends. I'd be fine renaming that across the board, but I think we should be consistent.

Reply:
:+1: